PCSamples/Graphics/SimpleHDR_PC/DeviceResources.cpp

```diff
@@ -398,6 +398,12 @@ bool DeviceResources::WindowSizeChanged(int width, int height)
     return true;
 }
 
+// This method is called when the Win32 display changes
+void DeviceResources::DisplayChanged()
```

Comment:
If you look at the linked PRs, I decided to make ``UpdateColorSpace`` a public method. That method first checks to see if the DXGI factory ``IsCurrent`` and if it is not, it recreates it.

This simplified Present() where it just calls ``UpdateColorSpace`` if ``IsCurrent`` return false. It also made it easier to handle ``WM_DISPLAYCHANGE`` without adding yet another 'pass-through' public method.